packages/compass-e2e-tests/helpers/commands/wait-for.ts

```diff
@@ -0,0 +1,24 @@
+import { promisify } from 'util';
+import type { CompassBrowser } from '../compass-browser';
+
+const wait = promisify(setTimeout);
+
+export async function waitFor<T>(
```

Comment:
Maybe add a comment that explains how this differs from [`waitUntil`](https://webdriver.io/docs/api/browser/waitUntil/)? Because this is, at first glance, not quite obvious to me :)

Reply:
Ah, forgot that this exists, I'll just switch to using this, thanks